Tasktive/Models/ViewModels/TasksViewModel.swift

```diff
@@ -167,6 +167,88 @@ final class TasksViewModel: ObservableObject {
         })
     }
 
+    func deleteTasks(by date: Date, indexSet: IndexSet) async -> Result<Void, UserErrors> {
+        let dateHash = getHashDate(from: date)
+        var mutableTask = tasks
+
+        indexSet.reversed().forEach { taskIndex in
```

Comment:
Why is it needed to reverse the indexSet?

Reply:
My idea is that when deleting multiple by indexes, you need to do it in reverse order.

```
arr = ['a', 'b', 'c', 'd', 'e']
indexes_to_remove = [2, 3]  # i want to remove 'c' and 'd'

# if you do it in order:
for index in indexes_to_remove:
    arr.pop(index)
# it will remove 'c' and 'e'

# but if you reverse
for index in reversed(indexes_to_remove):
    arr.pop(index)
# it will remove 'c' and 'd'
```

However, I don't sort index set. Maybe I'm overthinking here? 😆